examples/wasm/envoy-config.yaml

```diff
@@ -0,0 +1,91 @@
+static_resources:
+  listeners:
+  - name: main
+    address:
+      socket_address:
+        address: 0.0.0.0
+        port_value: 80
```

Comment:
make this port anything above 1024 (actually in 80xx space, 8000 should be good), so you don't need to be root to run the envoy command.

Reply:
Updated in this [commit](https://github.com/dengyijia/envoy-wasm/pull/1/commits/6cfcd9373cb609b95ec42ce16c6716fcb671a7a1)